tailwind.config.js

```diff
@@ -47,36 +60,89 @@ module.exports = {
         800: '#262626',
         900: '#171717'
       },
+      'dark-gray': {
+        0: '#000000',
+        50: '#171717',
+        100: '#262626',
+        200: '#404040',
+        300: '#525252',
+        400: '#737373',
+        500: '#A3A3A3',
+        600: '#D4D4D4',
+        900: '#FAFAFA'
+      },
       red: {
         50: '#FFF2F2',
         100: '#FFE6E6',
         200: '#FFBFBF',
         300: '#FF9999',
-        500: '#FF0000',
-        600: '#E60000'
+        500: '#ED1414',
+        600: '#CC0000',
+        700: '#A30000',
+      },
+      'dark-red': {
+        50: '#441411',
+        100: '#7D221C',
+        200: '#992923',
+        300: '#FFB5B0',
+        500: '#FF453A',
+        600: '#FF7D75',
+        700: '#BF342C'
       },
       orange: {
         50: '#FFFAF4',
         100: '#FFF5EA',
-        200: '#FFE5CA',
+        200: '#FFD2A3',
         300: '#FFD5A9',
         500: '#FF9629',
-        600: '#E68725'
+        600: '#E68725',
+        700: '#BF711F',
+      },
+      'dark-orange': {
+        50: '#382301',
+        100: '#7D4E05',
+        200: '#FFE7C2',
+        300: '#BF7708',
+        500: '#FF9F0A',
+        600: '#FFB94C',
+        700: '#FFD99D'
       },
       green: {
         50: '#F2FBF4',
         100: '#E6F7E8',
-        200: '#C0ECC6',
+        200: '#6FFD83',
         300: '#9AE1A4',
         500: '#02B31B',
-        600: '#02A118'
+        600: '#12BB41',
+        700: '#028614'
+      },
+      'dark-green': {
+        50: '#0B3812',
+        100: '#196925',
+        200: '#CCF5D2',
+        300: '#26A138',
+        500: '#32D74B',
+        600: '#70E381',
+        700: '#ADEFB7'
       },
       blue: {
         50: '#F7FBFE',
         100: '#EFF6FD',
         200: '#D6EAF9',
+        300: '#BDDDF5',
         500: '#5AA9E6',
-      }
+        600: '#5198CF',
+        700: '#3A6C92'
+      },
+      'dark-blue': {
+        50: 'rgba(76, 184, 255, 0.15)',
```

Comment:
We should use hex here right? Because it uses opacity also on tailwind

Reply:
updated